components/02-molecules/pop-out-image/_pop-out-image.scss

```diff
@@ -16,3 +16,7 @@
     margin-bottom: var(--size-spacing-7);
   }
 }
+
+.pop-out-image__clearfix {
```

Comment:
I see why we need a clearfix here. I'd be interested in hearing your thoughts but I typically set clearfixes with an `:after` to avoid any `:last-child` styles and to have a cleaner DOM.

Reply:
It's been so long since I've done a clearfix, I forgot how to do it. I originally tried as an `::after` and it didn't work, so I just gave up... I should have googled faster because I just needed to add `content: ''` to get it to work 🤦

Reply:
@kara-franco Can you verify that the `::after` pseudo element doesn't add anything unwanted to assistive tech? I saw some examples that used `visibility: hidden` but they all had a character in the "content" attribute, so I understood it for that... but since I'm using an empty content, I didn't think I would need it.

Can you just verify that though before I merge?

Reply:
There are no issues with having empty pseudo content! [Screen readers have mixed support](https://www.powermapper.com/tests/screen-readers/content/css-generated-content/) when it comes to interpreting pseudo content, so it's best to only use :after / :before for non-visual content (content that is hidden/empty) or decorative content. Even with decorative content, like icons, it's best to stick that in a `<span>` and use aria-hidden, since some screen readers do pick up CSS generated content (it's not stable though).